lib/mpl_toolkits/axisartist/tests/test_axislines.py

```diff
@@ -60,6 +62,7 @@ def test_Axes():
 @image_comparison(['ParasiteAxesAuxTrans_meshplot.png'],
                   remove_text=True, style='default', tol=0.075)
 def test_ParasiteAxesAuxTrans():
+    plt.rcParams.update({"xtick.direction": "in", "ytick.direction": "in"})
```

Comment:
```suggestion
    # Remove this line when this test image is regenerated.
    plt.rcParams.update({"xtick.direction": "in", "ytick.direction": "in"})
```
also for all the following image tests